packages/plugin-ext/src/main/browser/view/plugin-view-registry.ts

```diff
@@ -728,6 +729,10 @@ export class PluginViewRegistry implements FrontendApplicationContribution {
         }
     }
 
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    private shouldUpdateViewVisibility(clauseContext: any, e: any): boolean {
```

Comment:
The arguments here also shouldn't be `any`. Looks like `string|Set<string>` and `ContextKeyChangeEvent`.

Reply:
Fixed